client/components/ScenariosList/scenariosList.css

```diff
@@ -0,0 +1,16 @@
+.tm__scenario-entry {
+    border-style: solid;
+    border-width: 1px;
+    border-color: #888888;
+    margin-right: 1rem;
+    margin-bottom: 1rem;
+}
+
+.tm__scenario-entry:hover {
+    background-color: #f5f5f5;
+    transform: translate(1px, 1px);
```

Comment:
i'm not sure i like the visual of this punch-in effect on `:hover` cc @isaacdurazo for UX consideration

Reply:
@gnarf, @ktbee and I expressed the same sentiment

Reply:
Yup! @evmiguel , Corey and I talked about this and we think it will actually make more sense to have `transform: translate(1px, 1px);` on `active` state, since we already have a good visual indicator (background color change) for the `hover `state